lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -2573,7 +2646,7 @@ def main():
             gid=dict(type='str'),
             mode=dict(type='int'),
         )),
-        networks=dict(type='list', elements='str'),
+        networks=dict(type='list'),
```

Comment:
```suggestion
        networks=dict(type='list', elements='raw'),
```
That means that Ansible will just keep the values in the list as-is, and it shows that you didn't just forget to add `elements` here.